docs/blog/posts/2023/sr-linux-kubernetes-anycast-lab.md

```diff
@@ -0,0 +1,955 @@
+---
+date: 2023-08-10
+tags:
+  - kubernetes
+  - minikube
+  - sr linux
+  - containerlab
+  - metallb
+  - anycast
+authors:
+  - michelredondo
+---
+
+# SR Linux Kubernetes Anycast Lab
+
+[Containerlab](https://containerlab.dev/) is a game-changer for every network engineer. It helps you to effortlessly create complex network topologies and validate features, scenarios... At the same time, [Minikube](https://minikube.sigs.k8s.io/) unlocks the power of Kubernetes on your local machine to quickly test and experiment with containerized applications.
+
+Wouldn't it be great to combine both worlds?
+
+In this blog post we will explore a lab topology consisting of a Leaf/Spine [SR Linux](https://learn.srlinux.dev/) Fabric connected to a Kubernetes Cluster.
+
+Our k8s Cluster will feature [MetalLB](https://metallb.universe.tf/), which is a load-balancer implementation for bare metal clusters. This will unlock the possibility to have anycast services in our fabric.
```

Comment:
I think here we need to add some color to the lab goals. Right now it just says there is a lab with k8s and srl fabric, but what I'd like to see is what this lab gives to a reader.

I think it is important to mention that almost every k8s cluster leverages conventional DC fabric underneath to interconnect worker nodes together. Being able to test this combination is key for any network engineer working with a fabric supporting a k8s cluster.